packages/venia-concept/_buildpack/create.js

```diff
@@ -105,7 +104,8 @@ function createProjectFromVenia({ fs, tasks, options }) {
                 delete config.projects.venia;
                 fs.outputJsonSync(targetPath, config, { spaces: 2 });
             },
-            '{CHANGELOG*,LICENSE*,_buildpack/*}': tasks.IGNORE,
+            '{CHANGELOG*,LICENSE*,_buildpack,_buildpack/**,**/__tests__,**/__tests__/**}':
```

Comment:
This seems verbose because it is. We need to tell the matcher to ignore contents AND the top-level directory otherwise we end up with a bunch of empty `__tests__` folders :)